src/transformers/modeling_bert.py

```diff
@@ -408,8 +432,7 @@ def forward(
             attention_output = cross_attention_outputs[0]
             outputs = outputs + cross_attention_outputs[1:]  # add cross attentions if we output attention weights
 
-        intermediate_output = self.intermediate(attention_output)
```

Comment:
To solve the problem, my suggestions would be to wrap these two calls in a function `forward_chunk` which is part of this class (`def forward_chunk(self, ....)`) and call `apply_chunking_to_forward(self.chunk_size_feed_forward, self.seq_len_dim, self.forward_chunk, attention_output,)` here.

Reply:
Don't think I quite follow what you mean here. Which two calls do you want to wrap?
Did you mean to have a `forward_chunk` function in the `BertLayer` class?

Reply:
Ok I fixed it based on your input - looks ok to me now.

Reply:
Great, that's exactly what I meant :-)